src/apps/content-editor/src/app/components/Nav/ItemsFilter.js

```diff
@@ -1,23 +1,28 @@
-import { Search } from "@zesty-io/core/Search";
-import cx from "classnames";
-import { useSelector } from "react-redux";
-import styles from "./ContentNav.less";
+import TextField from "@mui/material/TextField";
+import InputAdornment from "@mui/material/InputAdornment";
+import SearchIcon from "@mui/icons-material/Search";
 
 const ItemsFilter = (props) => {
-  const ui = useSelector((state) => state.ui);
   return (
-    <Search
-      className={cx(
-        styles.SearchModels,
-        ui.contentNavHover ? "" : styles.HideIcon
-      )}
+    <TextField
       name="itemsFilter"
       placeholder="Filter items by name, zuid or path"
-      value={props.searchTerm}
```

Comment:
@agalin920 just double checking, since im now getting the term from `event` I dont need to transfer over `value={props.searchTerm}` right? This is all working with out value prop